File: careForEldersBack/patientBill/src/main/java/com/care4elders/patientbill/model/ServiceItem.java
// src/main/java/com/care4elders/patientbill/model/ServiceItem.java
package com.care4elders.patientbill.model;

import lombok.Data;


import org.springframework.data.annotation.*;;;


@Data
public class ServiceItem {
    @Id
    private String id;
    private String serviceName;
    private String description;
    private double rate;
    private int quantity;
    


}

File: careForEldersFront/src/app/models/service-item.ts
// src/app/billing/models/service-item.model.ts

export interface ServiceItem {
    id?: number;               // Optional ID for existing items
    serviceName: string;       // Name of the service (e.g., "Consultation")
    description: string;       // Detailed description
    rate: number;              // Price per unit
    quantity: number;          // Number of units
    amount?: number;           // Calculated total (rate * quantity)
  }
  
  // Optional: Utility functions for ServiceItem
 
  